deploy-ocp.yml

```diff
@@ -30,17 +30,17 @@
   hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: true
   tasks:
-    # Inject "start: false" in the layout to not start any VM yet.
-    # Starting the VM will be done later, either by the tool deploying
-    # OSP, or the one deploy RHOSO.
-    # VM initial configuration, when managed, is done using cloud-init.
-    - name: Ensure no VM is started when we create them during this run
+    # Set amount: 0 for all vms except the ocp ones.
+    # The rest are created by the create-infra playbook, and managing
+    # them from here can cause problems in some environments, e.g
+    # when running each playbook from a different zuul vm
+    - name: Ensure only ocp vms are managed here
       vars:
         _no_start: >-
           {% set _vms = {}                                          -%}
           {% for _type in _cifmw_libvirt_manager_layout.vms.keys()  -%}
```

Comment:
Using an internal var of the libvirt_manager in a playbook that just consumes the role is kind of a hack.